### PR TITLE
Misc style stuff

### LINKED ISSUE #305: layout issue for account drop down
Screen Shot 2020-07-01 at 1 55 42 PM

my window is 1270x1338px. firefox 77.0.1

### PR DESCRIPTION
**Pull request by lboeman**
Closes a few minor style issues.
820ae65 closes #305

### CODE REVIEW

**Comment by lboeman** [comment]
Here are the proposed fixes for the issues closed in this PR:
- Updated report recompute message to say "Scheduled report recompute successfuly."
- Page headers all use the same `<h2>` tag.
- Right justified the Account dropdown, I had to enforce a fixed width to get this to work so there may be wrapping for some users with really long emails.
justified_account_menu
- Added some space between select all/none buttons in column filters.
filter_spacing

File: sfa_dash/blueprints/reports.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def get(self, uuid):
         except DataRequestException as e:
             self.flash_api_errors(e.errors)
         else:
-            flash('Report recomputed successfully.', 'message')
+            flash('Scheduled report recompute successfully.', 'message')
```
Comment (wholmgren) [inline sfa_dash/blueprints/reports.py:282]: Should we add text along the lines of wait 30 seconds and refresh the page? Or add a timer? (like we have for the original waiting page)

Comment (lboeman) [reply sfa_dash/blueprints/reports.py:282]: The user should still land back on that page and be able to see that message. I believe there is also a script Tony added to automagically reload the page on a timer.

Comment (wholmgren) [reply sfa_dash/blueprints/reports.py:282]: I'm confused. This just adds a box to the previously rendered report. It doesn't send the user to another page.

Comment (lboeman) [reply sfa_dash/blueprints/reports.py]: Ah, I was thinking that the report status was reset to pending on recompute. Now I see that it continues to serve the old report until it gets recomputed. Updated the text.

**Pull request merged**